src/test-run/index.js

```diff
@@ -402,6 +411,11 @@ export default class TestRun extends Session {
         if (command.type === COMMAND_TYPE.assertion)
             return this._executeAssertion(command, callsite);
 
+        if (command.type === COMMAND_TYPE.setPageLoadTimeout) {
+            this.pageLoadTimeout = command.duration;
```

Comment:
we've already done it in `_adjustConfigurationWithCommand` method and even return null [here](https://github.com/DevExpress/testcafe/pull/1648/files#diff-a21f56bee759db22062e32250e3df48cR403)

Reply:
nice catch!